sql/parse/parse.go

```diff
@@ -1692,6 +1692,15 @@ func convertCreateTable(ctx *sql.Context, c *sqlparser.DDL) (sql.Node, error) {
 		}
 	}
 
+	// TODO: probably need to check for tiny, medium, and long
+	// gather all columns names that are blob type
+	blobCols := map[string]bool{}
+	for _, col := range c.TableSpec.Columns {
+		if col.Type.Type == "blob" {
+			blobCols[col.Name.String()] = true
+		}
+	}
```

Comment:
If you want to check for all of the types, just do a postfix check that ends with blob. `TINYBLOB`, `LONGBLOB`, etc. all end with blob and no other type does.